code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -291,18 +291,14 @@
 
 /obj/item/organ/hivelord_core/on_life()
 	..()
-	if(owner)
-		owner.adjustBruteLoss(-1)
-		owner.adjustFireLoss(-1)
-		owner.adjustOxyLoss(-2)
 	if(ishuman(owner))
 		var/mob/living/carbon/human/H = owner
 		CHECK_DNA_AND_SPECIES(H)
 		if(NOBLOOD in H.dna.species.specflags)
 			return
 
 		if(H.blood_volume && H.blood_volume < BLOOD_VOLUME_NORMAL)
-			H.blood_volume += 2 // Fast blood regen
```

Comment:
If you're going to nerf the blood regen you should remove the blood cost from the spawn function, otherwise it'll be garbage